### PR TITLE
Emonpi conf

### PR DESCRIPTION
**Pull request by upatricck**
Added the emonpi conf sensor to the rmc config

### CODE REVIEW

File: seshdash/models.py [hunk 1]
```diff
@@ -147,6 +147,32 @@ class Site_Measurements(models.Model):
     def __str__(self):
         return self.sesh_site.site_name
 
+
+class EmonPi(models.Model):
```
Comment (jlark) [inline seshdash/models.py]: Let's make this start with same prefix as other sensor sensor_emonpi

**Comment by jlark** [comment]
Thanks, I can't see where a emonpi is added by default to each site? This is more for the view when the user is creating a new site and would like to rename the field mapping.

**Comment by jlark** [comment]
Is this done?

**Comment by upatricck** [comment]
No, still adding the abilitiy to edity emonpi sensor on adding an rmc site.

**Pull request merged**